content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -45,7 +45,7 @@ provides a set of stateless replicas. Controllers such as
 
 * StatefulSet was a beta resource prior to 1.9 and not available in any Kubernetes release prior to 1.5.
 * The storage for a given Pod must either be provisioned by a [PersistentVolume Provisioner](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/persistent-volume-provisioning/README.md) based on the requested `storage class`, or pre-provisioned by an admin.
-* Deleting and/or scaling a StatefulSet down will *not* delete the volumes associated with the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
+* Deleting and/or scaling a StatefulSet down will *not* delete the persistent volumes claim objects created from any volumeClaimTemplates included in the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
```

Comment:
How about:
```suggestion
* If you delete a StatefulSet or make it have fewer replicas, your cluster will *not* delete any PersistentVolumeClaim objects that belonged to those removed replicas. This approach helps avoid deleting data that you still want to keep.
```
or some words along those lines?

Also, @medmedchiheb, you could if you wish mention a workaround / enhancement for anyone who does want to automate the deletion of those PVCs: use an appropriate Operator.